libmcount/mcount.c

```diff
@@ -1185,6 +1185,9 @@ int mcount_entry(unsigned long *parent_loc, unsigned long child,
 	struct mcount_ret_stack *rstack;
 	struct uftrace_trigger tr;
 
+	// store errno from target context.
+	int _saved_errno = errno;
+
```

Comment:
Is it always needed?  Or just for init time?  Also I think the comment is meaningless - it only says what it does not why.  Please just remove them.